packages/requisitions/src/CustomerRequisition/ListView/ListView.tsx

```diff
@@ -44,7 +47,24 @@ export const CustomerRequisitionListView: FC = () => {
   );
 
   const columns = useColumns<RequisitionRow>(
-    ['otherPartyName', 'requisitionNumber', 'status', 'comment', 'selection'],
+    [
+      getNameAndColorColumn((row: RequisitionRow, color: Color) => {
+        onUpdate({ ...row, color: color.hex });
+      }),
+      {
+        key: 'requisitionNumber',
+        label: 'label.number',
+      },
+      'status',
+      {
+        key: 'orderDate',
+        label: 'label.requisition-date',
+        width: 100,
+        accessor: rowData => (rowData.orderDate ? d(rowData.orderDate) : ''),
+      },
```

Comment:
I suppose since these are used in two places you could argue that defining them in the set builder file is better?  At least defined here then you know that `rowData` is a `RequisitionRow` and it's a bit type saferer

Reply:
more type safe and keeps the list of columns down a little